assets/sass/_hubspot.scss

```diff
@@ -91,7 +91,16 @@
 
     &.pricing {
         label {
-            @apply font-normal;
+            @apply font-normal text-base;
+        }
+        input, textarea {
+            @apply shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
+            &:focus {
+                @apply outline-none shadow-outline;
```

Comment:
I'm not sure I understand the intent with the borders and outlines. Is there some browser inconsistency you're solving for?

Reply:
Yup I am just making sure the focus outline is consistent across browsers. In my changes I updated the outline color to be purple to fit more in line with out theme. I'm interested to to get your opinion on this.